src/vs/editor/contrib/inlineCompletions/test/browser/utils.ts

```diff
@@ -131,3 +134,57 @@ export class GhostTextContext extends Disposable {
 		CoreEditingCommands.DeleteLeft.runEditorCommand(null, this.editor, null);
 	}
 }
+
+function generateRandomSimpleString(stringLength: number): string {
+	let randomText: string = '';
+	const possibleCharacters: string = ' abcdefghijklmnopqrstuvwxyz0123456789';
+	for (let i = 0; i < stringLength; i++) {
+		randomText += possibleCharacters.charAt(Math.floor(Math.random() * possibleCharacters.length));
```

Comment:
This does not include line breaks! Line breas are a crucial thing to test though

Reply:
Yes the line breaks I include in the second helper function `generateRandomMultilineString` which generates a random string spanning several lines.